### PR TITLE
Tests: improve unit tests coverage for the 2-phase Voting contract

### PR DESCRIPTION
**Pull request by TheDZhon**
### Implement 100% unit test coverage for the 2-phase Voting contract

image

### CODE REVIEW

File: apps/voting/test/voting.js [hunk 1]
```diff
@@ -217,18 +248,91 @@ contract('Voting App', ([root, holder1, holder2, holder20, holder29, holder51, n
...
+
+        it('canVote check works', async () => {
+          assert.equal(await voting.canVote(voteId, holder29), true, 'should be able to vote')
+          await voting.mockIncreaseTime(mainPhase + 1)
+          assert.equal(await voting.canVote(voteId, holder29), true, 'should be unable to vote')
+          await voting.mockIncreaseTime(objectionPhase)
+          assert.equal(await voting.canVote(voteId, holder29), false, 'should be unable to vote')
+        })
+
+        it('getVotePhase works', async () => {
```
Comment (folkyatina) [inline apps/voting/test/voting.js:309]: Can check phase in `getVote()` as well

**Review by folkyatina — COMMENTED** [review]
Only one minor notice and LGTM.

**Review by arwer13 — CHANGES_REQUESTED** [review]
LGTM 👍 , except for `unsafelyChangeVoteTime` lacking test for `auth(UNSAFELY_MODIFY_VOTE_TIME_ROLE)` authentication (sadly, it's not being displayed in the coverage report)

**Comment by TheDZhon** [comment]
The code has been updated, thank you for the findings :100: 

image

**Review by folkyatina — APPROVED** [review]

**Review by arwer13 — APPROVED** [review]

**Pull request merged**